test/parallel/test-http-agent-destroyed-socket.js

```diff
@@ -47,21 +47,9 @@ const server = http.createServer(common.mustCall((req, res) => {
     request1.socket.on('close', common.mustCall());
```

Comment:
Nit: I guess this is redundant now.

Reply:
Ah, good point. Will fix in the morning. Thanks!